internals/cli/cli.go

```diff
@@ -63,52 +63,34 @@ var optionsData options
 // ErrExtraArgs is returned  if extra arguments to a command are found
 var ErrExtraArgs = fmt.Errorf("too many arguments for command")
 
-// cmdInfo holds information needed to call parser.AddCommand(...).
-type cmdInfo struct {
-	name, shortHelp, longHelp string
-	builder                   func() flags.Commander
-	hidden                    bool
-	optDescs                  map[string]string
-	argDescs                  []argDesc
-	alias                     string
-	extra                     func(*flags.Command)
+// CmdInfo holds information needed to call parser.AddCommand(...).
+type CmdInfo struct {
```

Comment:
The field names here are a bit ambiguous, which would normally be fine but since we are making it public may be a short description to help people out.